rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -211,24 +210,6 @@ public Long getValue()
   @Override
   public List<BlockLocation> getCacheStatus(CacheStatusRequest request) throws TException
```

Comment:
We are no more using cluster_type from CacheStatusRequest, so the thrift idl and the users of CacheStatusRequest should be changed to not fill it.

Reply:
In a separate PR, I am removing all cluster_type reference where its not required.